api/envoy/config/filter/http/aws_lambda/v2/aws_lambda.proto

```diff
@@ -83,4 +83,8 @@ message AWSLambdaConfig {
     google.protobuf.Duration timeout = 3;
   }
 
+  // Send downstream path and method as `x-envoy-original-path` and
+  // `x-envoy-original-method` headers on the request to AWS lambda. 
+  // Defaults to false.
+  google.protobuf.BoolValue propagate_original_routing = 3;
```

Comment:
nit, this should be indented further left

Reply:
I think you may have been looking at an older version when reviewing as I unindented ~ the same time as this review. 
This is indented at 2 spaces in just like the oneof section declaration.
Unless I am missing something here?

Reply:
looks good to me now 👍 

yeah not sure what happened or what i saw there ¯\_(ツ)_/¯

Reply:
Do we care about a null value? If not this should be a primitive bool

Reply:
how confident are we in the default value here?

> Use wrapped scalar types if there is any potential need for a field to have a default value that does not match the proto3 defaults (0/false/""). For example, new features whose default value may change in the future or security mitigations that should be default safe in the future but are temporarily not enabled.

https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md

Reply:
I thought they were recommending it in general but in retrospect it makes sense as to that being the reason one would use a boolvalue rather than a straight bool.
Will change momentarily.

Reply:
Are we then good with keeping this as the primitive and have it be falsey by default? Thoughts?

Reply:
personally i think it should match xff header for default value and thus should also be `BoolVal` https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#envoy-v3-api-field-extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-use-remote-address

but i wont make a big stink about it if im outnumbered